rpmlb/cli.py

```diff
@@ -126,3 +135,15 @@ def run(recipe_file, collection_id, **option_dict):
     builder.run(work, **option_dict)
 
     log.info('Success!')
+
+
+def _adjust_option_dict(option_dict):
+    if not option_dict:
+        return
+    download = option_dict.get('download', None)
+    # When specifying download type: fedpkg or rhpkg,
+    # users must want to use the command in Builder.
```

Comment:
OK, it looks better. I will fix it.

Maybe below looks better.
Ref: http://docs.python-guide.org/en/latest/writing/style/#check-if-variable-equals-a-constant

```
# user chose the command explicitly
if not option_dict['pkg_cmd']:
    return
```